@ndlib/gatsby-theme-marble/src/assets/i18n/json/text.default.en.json

```diff
@@ -9,6 +9,11 @@
     "loginPrompt": "<0>Log in</0> to access and edit your own content.",
     "followPrompt": "<0>Log in</0> to follow users or access and edit your own content."
   },
+  "index": {
+    "nd": "University of Notre Dame",
+    "title": "Digital Collections",
+    "blockQuote": "Explore digitized rare materials from the University of Notre Dame, from books to manuscripts, sculptures to paintings, photographs to ephemera."
+  },
```

Comment:
The `IndexPage` lives completely in the `/site` directory. These translations should be in `/site/src/@ndlib/gatsby-theme-marble/assets/i18n/json/text.default.en.json`

Reply:
Move this to the correct file, then this looks good.